vector/src/main/java/im/vector/app/features/settings/devices/v2/rename/RenameSessionFragment.kt

```diff
@@ -89,7 +97,13 @@ class RenameSessionFragment :
                 title = getString(R.string.device_manager_learn_more_session_rename_title),
                 description = getString(R.string.device_manager_learn_more_session_rename),
         )
-        SessionLearnMoreBottomSheet.show(childFragmentManager, args)
+        SessionLearnMoreBottomSheet
+                .show(childFragmentManager, args)
+                .apply {
```

Comment:
I think in general in the project, we try to avoid using `apply` to avoid bugs on the scope of objects. Maybe it will be better to write it this way:
```
SessionLearnMoreBottomSheet
                .show(childFragmentManager, args)
                .onDismiss = { showKeyboard() }
```

Reply:
Done.